### PR TITLE
Upload chunk

### PR DESCRIPTION
**Pull request by AaronVasquez**

### CODE REVIEW

File: actions/upload_sessions.go [hunk 1]
```diff
@@ -82,6 +95,47 @@ func (usr *UploadSessionResource) Create(c buffalo.Context) error {
...
+
+			// Updates dmap in DB.
+			dm.Message = chunk.data
+			dm.Status = models.Unassigned
+			tx.ValidateAndSave(&dm)
+
+			dMaps[i] = dm
+		}
+
+		// Should we still do this here?
```
Comment (AaronVasquez) [inline actions/upload_sessions.go]: @rfornea Do we still want to process chunks here, or just let the cron job take care of this?

File: actions/upload_sessions.go [hunk 2]
```diff
@@ -82,6 +95,47 @@ func (usr *UploadSessionResource) Create(c buffalo.Context) error {
...
+		dMaps := make([]models.DataMap, len(req.chunks))
+		for i, chunk := range req.chunks {
+			// Fetch DataMap
+			dm := models.DataMap{}
+			tx.RawQuery(
+				"SELECT * from data_maps WHERE genesis_hash ? AND chunk_idx = ?", uploadSession.GenesisHash, chunk.idx).First(&dm)
+
+			// Updates dmap in DB.
+			dm.Message = chunk.data
+			dm.Status = models.Unassigned
```
Comment (AaronVasquez) [inline actions/upload_sessions.go]: @rfornea Is `unassigned` the correct status here?

Comment (rfornea) [reply actions/upload_sessions.go]: I believe so.  As long as it has the message we should be good to go, address and message is what we need.

File: actions/upload_sessions.go (hunk 1 above)
Comment (rfornea) [reply actions/upload_sessions.go]: We can pick it up in the cron job.  We can change the frequency of the cron task to run as often as we like.

**Comment by rfornea** [comment]
I'm just gonna go ahead and approve and I'll pull those two lines out in my next PR.

**Pull request merged**